libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -659,30 +668,40 @@ namespace SohImGui {
                 EnhancementSliderInt("King Zora Speed: %dx", "##WEEPSPEED", "gMweepSpeed", 1, 5, "");
 
                 EnhancementCheckbox("Skip Text", "gSkipText");
+                Tooltip("Holding down B skips text");
                 EnhancementCheckbox("Minimal UI", "gMinimalUI");
+                Tooltip("Hides most of the UI when not needed");
                 EnhancementCheckbox("MM Bunny Hood", "gMMBunnyHood");
+                Tooltip("Wearing the Bunny Hood grants a speed increase like in Majora's Mask");
                 EnhancementCheckbox("Visual Stone of Agony", "gVisualAgony");
+                Tooltip("Displays an icon and plays a sound when Stone of Agony should be activated, for those without rumble");
 
                 ImGui::Text("Graphics");
                 ImGui::Separator();
 
                 EnhancementCheckbox("N64 Mode", "gN64Mode");
+                Tooltip("Sets aspect ratio to 4:3 and lowers resolution to 240p, the N64's native resolution");
 
                 EnhancementCheckbox("Animated Link in Pause Menu", "gPauseLiveLink");
                 EnhancementCheckbox("Enable 3D Dropped items", "gNewDrops");
                 EnhancementCheckbox("Faster Block Push", "gFasterBlockPush");
                 EnhancementCheckbox("Dynamic Wallet Icon", "gDynamicWalletIcon");
+                Tooltip("Changes the rupee in the wallet icon to match the wallet size you currently have");
                 EnhancementCheckbox("Always show dungeon entrances", "gAlwaysShowDungeonMinimapIcon");
+                Tooltip("Alays shows entrances to dungeons on the minimap");
```

Comment:
Typos there :D
"alays"
should be "Always"
also might be me but the English seem off ?
Wouldn't  ``Always show dungeons entrances icon on the minimap`` be better ? (might be me there)

Reply:
Noticed the typo as I was fixing the other one! 
Both ways of saying it seem correct to me, yours is a little clearer though.